{{ cookiecutter.package_name|slugify }}/docker-compose.yml

```diff
@@ -12,6 +12,8 @@ services:
         {%- endif %}
       context: .
       target: dev
+    tty: true
+    entrypoint: []
```

Comment:
Alternatively: set the entrypoint like this `entrypoint: ["sh", "-c"]`, then you can simplify the `command` entry.

Reply:
Good suggestion!

Reply:
Actually, on second thought it's not that easy: the entrypoint needs to be an executable, which means the command would still need to start with `["-c", "cp ..."]`, which is IMHO less clear than the way it is now.